src/core_plugins/kibana/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "CORE-WELCOME_MESSAGE": "<strong>Kibana</strong> <small>is loading. Give me a moment here. I'm loading a whole bunch of code. Don't worry, all this good stuff will be cached up for next time!</small>",
```

Comment:
@Bargs, it was a review [comment](https://github.com/elastic/kibana/pull/7545#r72890673) from @srl295